es-ds-components/components/es-support-card.vue

```diff
@@ -0,0 +1,150 @@
+                    :class="{
+                        'pl-lg-100': constrained,
+                        'pl-lg-200': !constrained,
+                    }">
+                    <slot name="headline" />
+                </h2>
+                <div class="EsSupportCard-imageContainer mb-150 mb-lg-0 order-0">
+                    <slot name="image">
+                        <nuxt-img
+                            v-if="imageAltText && imageSrc"
+                            :alt="imageAltText"
+                            class="bg-teal-200 rounded-circle"
+                            height="100px"
+                            :src="imageSrc"
+                            width="100px" />
+                    </slot>
+                </div>
+                <!--
+                    we are giving 'EsSupportCard-description' a unique class (that we don't use directly in this
+                    component) for consuming applications to use in targeting ::v-deep styles on <p> tags to remove
+                    the natural <p> bottom margin, for example.
+
+                    e.g. a CMS rich text component may generate a <p> tag within this that automatically
+                    gets a bottom margin, which throws off the vertical centering. we are intentionally leaving
+                    the removal of that bottom margin up to consuming applications rather than removing it on all
+                    <p> tags within this element, in case they may want two <p> tags or any other markup in here.
+                -->
+                <div
+                    class="EsSupportCard-description font-size-75 font-size-lg-100 mb-150 mb-lg-0 order-2"
+                    :class="{
+                        'pl-lg-100': constrained,
+                        'pl-lg-200': !constrained,
+                    }">
+                    <slot name="description" />
+                </div>
+            </b-col>
+            <b-col
+                cols="12"
+                lg="4"
+                xxl="3">
+                <es-button
+                    class="w-100"
+                    :class="{ 'mb-100': showSecondaryCta }"
+                    :href="primaryCtaUrl"
+                    :target="primaryCtaTarget"
+                    variant="primary">
+                    {{ primaryCtaText }}
+                </es-button>
+                <es-button
+                    v-if="showSecondaryCta"
+                    class="w-100"
+                    :href="secondaryCtaUrl"
+                    outline
+                    :target="secondaryCtaTarget"
+                    variant="primary">
+                    {{ secondaryCtaText }}
+                </es-button>
+            </b-col>
+        </b-row>
+    </es-card>
+</template>
+
+<style lang="scss">
+@use '@energysage/es-ds-styles/scss/mixins/breakpoints' as breakpoints;
+
+@include breakpoints.media-breakpoint-up(lg) {
+    .EsSupportCard {
```

Comment:
Can we move these styles to a SASS file in es-ds-styles so all component styles live in there? Perhaps in the `/scss/vue/components/` folder.

Reply:
We haven't really standardized this, but I do think there's an argument for keeping them in the component as well, particularly if we're able to make it `scoped`. It will reduce the size of the CSS bundle that gets served on every page.

Reply:
I've generally kept them in the component unless there's a ton of styles and I'm not able to make it scoped

Reply:
That's interesting, and worth a discussion. If we want the bundle to be smaller, that'd end up with us putting as much of the component CSS in the component files themselves, which would eventually result in es-ds-styles only containing global styles.

Since it looks like we have inconsistency already on this, @hroth1994 I'd say it's fine to leave the way you have it.

Reply:
> If we want the bundle to be smaller, that'd end up with us putting as much of the component CSS in the component files themselves, which would eventually result in es-ds-styles only containing global styles.

Yeah honestly the more I think about it, the more this seems like the right approach. We can reduce the global CSS bundle size, and prevent people from relying on CSS classes that are actually tailored to a specific component implementation and subject to change.